client/blocks/cs-embed/sidebar.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { Button, PanelBody, PanelRow } from '@wordpress/components';
+import { InspectorControls } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import SidebarPromote from 'components/sidebar-promote';
+
+const Sidebar = ( { attributes, shouldPromote, signalWarning } ) => {
+	const { typeTextPlural, createText, dashboardLink } = attributes;
+	return (
+		<InspectorControls>
+			<PanelBody
+				title={ __( 'Crowdsignal Settings', 'crowdsignal-forms' ) }
+				initialOpen={ true }
+			>
+				<div>
+					Edit your { typeTextPlural } on{ ' ' }
+					<a
+						href={ dashboardLink }
+						target="_blank"
+						rel="external noreferrer noopener"
+					>
+						Crowdsignal.com
+					</a>
```

Comment:
When creating a concatenated string, to make it translatable, there is this helper [createInterpolateElement](https://github.com/WordPress/gutenberg/tree/trunk/packages/element#createInterpolateElement) we need to use.
If you need some examples on how to use it, you can find some [here](https://github.com/search?q=org%3AAutomattic+createInterpolateElement&type=code).

Another thing I didn't understand well is why we need the `typeTextPlural` as an attribute/variable. 
Couldn't we just inline the string?